components/secret-mgt/org.wso2.carbon.identity.secret.mgt.core/src/main/java/org/wso2/carbon/identity/secret/mgt/core/SecretManager.java

```diff
@@ -124,4 +124,9 @@
      * @throws SecretManagementException Configuration Management Exception.
      */
     void deleteSecretType(String secretTypeName) throws SecretManagementException;
+
+    Secret updateSecretValue(String secretTypeName, String name, String value) throws SecretManagementException;
```

Comment:
Allowing the addition of the new methods to the interface because this is added recently and we don't have any public release with this interface.

Reply:
We need method doc comments for these new methods.

Reply:
resolved